apiservermsgs/pgadminmsgs.go

```diff
@@ -0,0 +1,103 @@
+package apiservermsgs
+
+/*
+Copyright 2020 Crunchy Data Solutions, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import ()
+
```

Comment:
I think this gets removed with `gofmt -s` which is what [goreportcard](https://goreportcard.com/report/github.com/CrunchyData/postgres-operator) runs